### PR TITLE
added new line

### PR DESCRIPTION
**Pull request by iBerq**
added new print line

### CODE REVIEW

File: file.py [hunk 1]
```diff
@@ -4,3 +4,4 @@ def sum(a, b):
 
 print(sum(2, 2))
 print(sum(3, 4))
+print(sum("a", "b"))
```
Comment (pelinceliksoz) [inline file.py:7]: what is a and b? There is a problem here

**Review by pelinceliksoz — CHANGES_REQUESTED** [review]
check the problem

File: file.py (hunk 1 above)
Comment (iBerq) [reply file.py:7]: will look into it

File: file.py [hunk 2]
```diff
@@ -4,4 +4,4 @@ def sum(a, b):
 
 print(sum(2, 2))
 print(sum(3, 4))
-print(sum("a", "b"))
+print(sum(5, 6))
```
Comment (pelinceliksoz) [inline file.py:7]: better

**Review by pelinceliksoz — COMMENTED** [review]
All done!

File: file.py (hunk 2 above)
Comment (iBerq) [reply file.py:7]: thank you

**Pull request merged**